pkgs/development/python-modules/nameko/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+
+# install_requires
+, eventlet
+, kombu
+, mock
+, packaging
+, path
+, pyyaml
+, requests
+, setuptools
+, six
+, werkzeug
+, wrapt
+}:
+
+buildPythonPackage rec {
+  pname = "nameko";
+  version = "2.14.1";
+  format = "setuptools";
```

COuld you please check if PEP 518 is supported?

```suggestion
  pyproject = true;
```

and add `setuptools` as `nativeBuildInputs`.